index.html

```diff
@@ -494,7 +563,7 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.16
               <li>Known issues:</li>
               <ol>
                   <li><a href="https://github.com/pcengines/apu2-documentation/issues/308">APU7 iPXE network boot with i225 NICs does not work</a></li>
-                  <li><a href="https://github.com/pcengines/coreboot/issues/329">apuled driver doesn't work in FreeBSD.</a> Check the <a target="_blank" href="https://github.com/pcengines/apu2-documentation/blob/master/docs/gpios.md">GPIOs</a> document for workaround.</li>
+                  <li><a href="https://github.com/pcengines/coreboot/issues/329">apuled driver doesn't work in FreeBSD.</a> Check the <a target="_blank" href="https://github.com/pcengines/apu2-documentation/blob/master/docs/gpios.md"> GPIOs</a> document for workaround.</li>
```

Comment:
Please revert this unnecessary change

Reply:
@miczyg1 this point have been changed, I add `&nbsp;` and space before `GPIOs` and `mPCIe`. Only with such a declaration the space in between two word is visible (I rely on locally built documentation).